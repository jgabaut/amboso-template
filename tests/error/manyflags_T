#!/bin/bash

"./anwrap" -qTibprdgB "0.1.0"
