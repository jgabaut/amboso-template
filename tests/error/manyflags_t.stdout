^[[1;36m[DEBUG]    !NO! Backtrace ["set AMBOSO_TRACING="].^[[0m$
$
^[[1;35mTry uncommenting line 10 in "anwrap" to have some backtrace. Very WIP.^[[0m$
^[[1;31m[DEBUG]    Recording all tests with -ti is deprecated.$
$
        Feature will be dropped in next major update.$
$
$
[AMBOSO]    Exceeded depth for recursion ( nested 3 times).$
$
$
[AMBOSO]    Exceeded depth for recursion ( nested 3 times).$
$
$
[AMBOSO]    Exceeded depth for recursion ( nested 3 times).$
$
$
[AMBOSO]    Exceeded depth for recursion ( nested 3 times).$
$
$
[AMBOSO]    Exceeded depth for recursion ( nested 3 times).$
$
$
[AMBOSO]    Exceeded depth for recursion ( nested 3 times).$
$
$
[AMBOSO]    Exceeded depth for recursion ( nested 3 times).$
$
